src/components/UcrParticipationInformation.js

```diff
@@ -58,11 +54,13 @@ const UcrParticipationInformation = ({ dispatch, place, timeTo, ucr }) => {
             </Term>
           )} data to the FBI.
         </p>
-        {/* eslint max-len: 0 */}
```

Comment:
I think we should stick with this sort of pattern/linting config for large content blocks so that the content folks can just edit it without having to worry about line length.